### ISSUE TITLE
String Upper for keyword identifier

### ISSUE THREAD

**Issue by jlkweb12** [yajra/laravel-oci8#350]
### Summary of problem or feature request

In the 5.5 version, forcing upper case is breaking databases.

### Code snippet of problem

This was changed from in the method OracleGrammar->wrapValue:
`$value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);`

to:
`$value = Str::upper($value);`

This is causing migration issues. I have multiple systems talking to the database so I need to leave it lowercase.  If this required, how can we get around this change. This is a break change from 5.4 to 5.5.

### System details

- Ubuntu 16.04
- PHP Version 7.1
- Laravel Version 5.5
- Laravel-OCI8 Version 5.5

**Comment by yajra** [comment]
Duplicate of #341, unfortunately you have to manually update the reserved words to uppercase when upgrading to 5.5. Thanks!

**Comment by jlkweb12** [comment]
Then I need a different part of the code updated to work with this change.

In Schema->OracleGrammar->wrapValue:

```
if ($this->isReserved($value)) {
            return Str::lower(parent::wrapValue($value));
        }
```

This is causing the seeder to break. Can we get that to move to upper then?

**Comment by yajra** [comment]
@jlkweb12 good catch. Will released a fix in a while.

**Comment by yajra** [comment]
Patch released on v5.5.3, kindly update and confirm. Thanks!

**Comment by jlkweb12** [comment]
It works. Thanks for the quick turn around.
